components/jaggery-core/org.jaggeryjs.jaggery.core/src/main/java/org/jaggeryjs/jaggery/core/manager/CommonManager.java

```diff
@@ -301,35 +306,47 @@ public static void print(Context cx, Scriptable thisObj, Object[] args, Function
         OutputStream out = (OutputStream) jaggeryContext.getProperty(CommonManager.JAGGERY_OUTPUT_STREAM);
         if (args[0] instanceof StreamHostObject) {
```

Comment:
Yes. Since this is a patch fix, code improvements for the CommonManager is not applied. Fixed with 605e3309b45c85ae227e77ef4c30a8121aabc480.